dts/bindings/syscon/bflb,efuse.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024-2025 MASSDRIVER EI (massdriver.space)
+# SPDX-License-Identifier: Apache-2.0
+
+description: BouffaloLab Efuse Driver
+
+compatible: "bflb,efuse"
```

Comment:
```diff
-compatible: "bflb,efuse"
+compatible: "bflb,sysconn-efuse"
```

Reply:
I don't have strong opinion about this but since we are using another driver API to implement the functionality that I expect when I read the compatible with sysconn I can not identify that. So, I'll not block this.

Reply:
It's not my first choice anyway, syscon is notably less practical than eeprom for this but it's what i was ordered to use.

Either way finding binding folder from names doesnt work most of the time, `grep -r <binding name>` is my method.

Reply:
No worries, I'll not block this.